tests/cross-compile/fmod.slang

```diff
@@ -4,7 +4,8 @@
 // expected output on Vulkan/GLSL.
 
 //TEST(compute):COMPARE_COMPUTE:-dx11 -compute -shaderobj
-//TEST(compute):COMPARE_COMPUTE:-vk -compute -shaderobj
+//TEST(compute):COMPARE_COMPUTE:-vk -compute -shaderobj -emit-spirv-via-glsl
```

Comment:
Just `//TEST(compute):COMPARE_COMPUTE:-vk -compute -shaderobj`.
CI will run with both config.

Reply:
Right. But I was little confused when my local tests are all passing and only CI was failing.